spec/defines/windows_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'spec_helper'
+describe 'local_user::windows', :type => :define do
+  let (:title) { 'rnelson0' }
+  let (:params) do
+    {
+      :state    => 'present',
+      :groups   => [],
+      :password => 'Microsoft1',
+    }
+  end
+
```

Comment:
We need to set the default fact of the kernel as Windows, and add a test case where the fact is overriden to Linux. That test case should fail.